src/components/sidebar/Sidebar.jsx

```diff
@@ -30,29 +30,32 @@ export const Sidebar = ({ dispatch, state}) => {
                        
                     </div>
 
+
                     <div className="category-filter">
-                        <h3>Price Range</h3>
+                        <h3>Brand</h3>
 
-                        {/* ----------------------filter by price range ------------------- */}
-                        <div>
-                            <input type="checkbox" id="range-1000" />
-                            <label htmlFor="range-1000">1000- 1500</label>
+                         {/* -----------------------filter by brand -------------- */}
+
+                         <div>
+                            <input type="checkbox" className="input-checkbox" id="nike-collection" checked={state.Nike} onChange={() => dispatch({type: "NIKE_BRAND"})} />
+                            <label htmlFor="nike-collection">Nike</label>
                         </div>
-                        <div>
```

Comment:
these commented line is really appreciation:
It helps to readability while reviewing